frontend/main-app/src/components/DeleteForm/DeleteForm.tsx

```diff
@@ -0,0 +1,88 @@
+import classes from "./DeleteForm.module.css";
+import { Button, Modal, Paragraph, Textfield } from "@digdir/design-system-react";
+import React, { FormEvent, ReactNode, useContext, useState } from "react";
+import { TrashFillIcon } from "@navikt/aksel-icons";
+import { FormContext } from "../Dashboard";
+import { getApiUrl } from "../Login/LoginUtils";
+import axios from "axios";
+import { Trans } from "react-i18next";
+import { t } from "i18next";
+
+interface ModalContentProps {
+  className: string;
+}
+
+const ModalContent: React.FC<ModalContentProps> = ({ className }) => {
```

Comment:
I think this component can be refactured in a seperat issue later , by moving ModalContent to an own componenrt.

Reply:
I did a quick refactor just now - can you check it out in the latest commits?
![image](https://github.com/ErlingHauan/FormFactory/assets/148075168/21374fb4-7da4-4fca-a91c-e0bd9d00e6bd)